Source/WebKit/Shared/Site.h

```diff
@@ -38,6 +38,10 @@ class Site {
     Site(const Site&) = default;
     Site& operator=(const Site&) = default;
 
+    Site(String&&, WebCore::RegistrableDomain&&);
```

Comment:
We should not omit the first parameter name, it is not obvious it is a protocol.

Reply:
This is removed.